medical/data/medical_tour.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Make the xmlid of menus required by the Medical tour available in webclient -->
+    <record id="medical.medical_root" model="ir.ui.menu">
```

Comment:
Any idea why lint is complaining about the xml_id here being redundant? Seems like something we can ignore, just seemed odd.